### PR TITLE
[CoSim] Add MPM DEM coupling 

### PR DESCRIPTION
**Pull request by VeronikaSinger**
Hi all,

this PR adds the MPM-DEM coupling. 
In the CoSim App an additional wrapper for the Neumann Interface is added as well as a MPM-DEM coupling test.
In the ParticleMechanicsApp a condition process is added to create the Neumann interface and the particle point load condition is updated. 

test_MPM-DEM_Coupling (77)

### CODE REVIEW

**Comment by VeronikaSinger** [comment]
Hi @KratosMultiphysics/cosimulation @KratosMultiphysics/mpm are there any suggestions? Can we merge this?

**Comment by VeronikaSinger** [comment]
@philbucher are there any suggestions from your side? :)

File: applications/CoSimulationApplication/tests/mpm_fem_beam/ProjectParametersMPM.json [hunk 1]
```diff
@@ -39,14 +39,11 @@
             "scaling": false
         }
     },
-    "coupling_settings": {
```
Comment (philbucher) [inline applications/CoSimulationApplication/tests/mpm_fem_beam/ProjectParametersMPM.json:42]: Nice cleanup

**Review by philbucher — APPROVED** [review]
Nice, good work

No comments, everything looks good to me 👍🏻

Sorry for the delay

**Comment by VeronikaSinger** [comment]
@philbucher thanks for the review.

**Pull request merged**